tools/tune/drc/drc_gen_coefs.m

```diff
@@ -29,6 +29,7 @@
 % Calculate makeup gain coefficients
 full_range_makeup_gain = (1 / coefs.ratio_base) ^ 0.6; % Empirical/perceptual tuning
 coefs.master_linear_gain = db2mag(params.post_gain) * full_range_makeup_gain;
+coefs.master_linear_gain_db = 20*log10(coefs.master_linear_gain);
 
```

Comment:
why do we add master_linear_gain_db?  seems we only use master_linear_gain when processing.

Reply:
following code seems does not use this db.

Reply:
I added it to know the dB gain achieved with all the parameters. Due to above empirical/perceptual tuning commented equation it can't be directly seen from the parameters input. I need to add some print to make this used. I guess I checked it with debugger or print of the same code line with `;` removed.

Reply:
Sorry, the master_linear_gain_db is actually printed in line 66, so it is used.

```
% Print out coefs
coefs

```